permissions/plugin-artifact-manager-s3.yml

```diff
@@ -7,3 +7,6 @@ developers:
 - "jglick"
 - "csanchez"
 - "oleg_nenashev"
+security:
+  contacts:
+    jira: "cloud_platform_v2"
```

Comment:
Not sure exactly what this is, but if it goes to at least @carlossg then that is fine.

Reply:
@jglick carlossg has been added to the distro list